react/week3/todos/src/components/todolist.js

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+
+class Form extends React.Component {
+    render() {
+      return (
+        <div>
+          <form onSubmit={this.props.handleSubmit}>
+            <label>Todo description:
+              <input value={this.props.description} onChange={this.props.handleDescriptionChange} name="description" required /><br />
+            </label>
+            <label>Deadline:
+              <input type="date" value={this.props.deadline} onChange={this.props.handleDeadlineChange} name="deadline" required /><br />
+            </label>
+            <input type='submit' value='Add todo' />
```

Comment:
I would strongly advise against using an input as a submit button. Although it's clearly possible, it's not always clear whats going on in your components: 
```suggestion
           <button type="submit">Add todo</button>
```